api/openapi/v1/support-notifications.yaml

```diff
@@ -22,21 +31,21 @@ paths:
                 $ref: '#/components/schemas/Error'
   /cleanup/age/{age}:
     delete:
-      description: Delete all the Notifications if the current timestamp minus their
+      description: Delete all the notifications if the current timestamp minus their
         last modification timestamp is less than the age parameter, and the corresponding
-        Transmissions will also be deleted.
+        transmissions will also be deleted.
       parameters:
       - name: age
         in: path
-        description: Specify the age of <<typeName>>, and the format is in milliseconds.
+        description: Specify the age of parameter, with the format in milliseconds.
```

Comment:
"specify the age of parameter" doesn't make sense, and I think the comma breaks up the sentence in an awkward way.